likecoin/views/metabox.php

```diff
@@ -1,61 +1,92 @@
+	</div>
+	<div class="likecoin webThreeError needMainNet" style="display: none">
+	<h3>Please switch to Main Network</h3>
+	</div>
+	<div class="likecoin webThreeError needUnlock" style="display: none">
+	<h3>Please unlock your wallet</h3>
+	</div>
+	<div class="likecoin webThreeError needLikeCoinId" style="display: none">
+	<h3>Need Like Coin Id</h3>
+	</div>
+	<div class="likecoin webThreeError needLogin" style="display: none">
+	<h3><a class="loginBtn" style="cursor: pointer">Login to get Like Coin Id</a></h3>
+	</div>
 </section>
 <section class="likecoin optionsSection" style="<?php echo $has_likecoin_id ? '' : 'display: none'; ?>">
-  <div style="display:flex">
-    <section>
-      <div id="updateLikeCoinIdStatus"></div>
-      <div>LikeCoin Id: <span id="likecoinId"><?php echo sanitize_text_field($likecoin_id); ?></span>
-        <a class="changeBtn" style="cursor: pointer">change</a></div>
-      <div>LikeCoin Wallet: <span id="likecoinWallet"><?php echo sanitize_text_field($likecoin_wallet); ?></span></div>
-      <label for="lc_widget_option">Display Option</label>
-      <select name="lc_widget_option" id="lc_widget_option" class="postbox">
-          <option value="both" <?php if ( $widget_position === 'both' ) echo 'selected'; ?>>Top and bottom</option>
-          <option value="top" <?php if ( $widget_position === 'top' ) echo 'selected'; ?>>Top</option>
-          <option value="bottom" <?php if ( $widget_position === 'bottom' ) echo 'selected'; ?>>Bottom</option>
-          <option value="none" <?php if ( $widget_position === 'none' ) echo 'selected'; ?>>None</option>
-      </select>
-    </section>
-    <section style="flex:1;-webkit-flex:1;-ms-flex:1">
-      <iframe id="likecoinPreview" scrolling="no" frameborder="0"
-        style="pointer-events: none; height: 212px; width: 100%;"
-        src="<?php if ( $has_likecoin_id ) echo 'https://button.like.co/in/embed/' . sanitize_text_field($likecoin_id) . '/button'; ?>"
-      ></iframe>
-    </section>
-  </div>
+	<div style="display:flex">
+	<section>
+		<div id="updateLikeCoinIdStatus"></div>
+		<div>LikeCoin Id: <span id="likecoinId"><?php echo esc_html( $likecoin_id ); ?></span>
+		<a class="changeBtn" style="cursor: pointer">change</a></div>
+		<div>LikeCoin Wallet: <span id="likecoinWallet"><?php echo esc_html( $likecoin_wallet ); ?></span></div>
+		<label for="lc_widget_option">Display Option</label>
+		<select name="lc_widget_option" id="lc_widget_option" class="postbox">
+			<option value="both" 
+			<?php
+			if ( 'both' === $widget_position ) {
+				echo 'selected';}
+			?>
+			>Top and bottom</option>
+			<option value="top" 
+			<?php
+			if ( 'top' === $widget_position ) {
+				echo 'selected';}
+			?>
+			>Top</option>
+			<option value="bottom" 
+			<?php
+			if ( 'bottom' === $widget_position ) {
+				echo 'selected';}
+			?>
+			>Bottom</option>
+			<option value="none" 
+			<?php
+			if ( 'none' === $widget_position ) {
+				echo 'selected';}
```

Comment:
The `}` at the end of lines (instead of in a new line) are the recommended PHP style?

Reply:
seems phpcs autofix doesnt care, will change to newline